src/main/java/com/beust/jcommander/DefaultUsageFormatter.java

```diff
@@ -183,6 +183,7 @@ public void appendAllParametersDetails(StringBuilder out, int indentCount, Strin
                     .append("  ")
                     .append(parameter.required() ? "* " : "  ")
                     .append(pd.getNames())
+                    .append(parameter.placeholder() != null && !parameter.placeholder().isBlank() ? " " + parameter.placeholder() : "")
```

Comment:
* Feel free to shorten as `.append(parameter.placeholder() instanceof String placeholder && !placeholder.isBlank() ? " " + placeholder : "")` if you like. We are on Java 17 in this branch. 😃 

* IIUC `parameter.placeholder()` *cannot* be `null` as it is defined as `String default ""`).

Reply:
2nd point: Yes, I originally had this as null, then I noticed that your convention for unset string parameters was to have them default to the empty string. I'll remove the null check.

Reply:
Actually `./gradlew build` is defaulting to something earlier than JDK 17 (I'm on JDK 23), and I don't want to mess with the build files, so I'm not including the JDK 17 suggestion.